semana10/projeto-futureX/src/containers/FormPage/index.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { connect } from "react-redux";
+import { push, replace, goBack } from "connected-react-router";
+import { routes } from "../Router";
+import styled from "styled-components";
+
+const MainContainer = styled.div`
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+`
+
+const Header = styled.header`
+  width: 100%;
+  height: 60px;
+  background: green;
+`
+
+const Footer = styled.footer`
+  width: 100%;
+  height: 60px;
+  background: green;
+  position: absolute;
+  bottom: 0;
+`
+
+class FormPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            postValue: "" //lembrar de mudar nome disso aqui
```

Comment:
Dica: Quando for colocar lembretes de coisas a serem feitas no código comece o comentário com a palavra TODO:

```js
// TODO lembrar de mudar nome disso aqui
```

Assim, ao final do projeto, basta você procurar por TODO para ver pendências.